store/sqlstore/emoji_store.go

```diff
@@ -193,51 +166,5 @@ func (es SqlEmojiStore) getBy(what string, key interface{}, addToCache bool) (*m
 		return nil, model.NewAppError("SqlEmojiStore.GetByName", "store.sql_emoji.get.app_error", nil, "key="+fmt.Sprintf("%v", key)+", "+err.Error(), status)
 	}
 
-	if addToCache {
-		es.addToCache(emoji)
-	}
-
 	return emoji, nil
 }
-
-func (es SqlEmojiStore) addToCache(emoji *model.Emoji) {
-	emojiCacheById.AddWithExpiresInSecs(emoji.Id, emoji, EMOJI_CACHE_SEC)
-	emojiIdCacheByName.AddWithExpiresInSecs(emoji.Name, emoji.Id, EMOJI_CACHE_SEC)
-}
-
-func (es SqlEmojiStore) getFromCacheById(id string) (*model.Emoji, bool) {
-	if cacheItem, ok := emojiCacheById.Get(id); ok {
-		es.incrementMemCacheHitCounter("Emoji")
-		return cacheItem.(*model.Emoji), true
-	}
-	es.incrementMemCacheMissCounter("Emoji")
```

Comment:
You have removed this kind of `metrics` calls used to measure the cache hits/misses, do you mind to add them again?

Reply:
These metrics where removed because the doStandardReadCache function is already increasing the metrics. Should I put them back?

Reply:
Good point, you are right.